### DISCUSSION TITLE
Conditional Tags

### DISCUSSION THREAD

**Discussion by MichaelWybo** [liedekef/events-made-easy#507] [General]
Dag Franky

I try to show list of next events, 
Linking to the event details, but the text should be different if the logged in user is booked and approved, on waiting list or can book

```
[eme_if2 tag='#_IS_USER_IN_GROUP{Chiots}' value='1']

Cours Chiots
[eme_events scope=+15d limit=0 template_id=12 category=4][/eme_if2]

```
So i use these conditional tags in the template '12':

```
<li>#_STARTDATE{d/m/Y H:i} - #_LINKEDNAME - <strong>
[eme_if tag='#_IS_REGISTERED_APPROVED' value='1']<span class="cancellink"><a href="#_EVENTPAGEURL">Inscrit</a></span>[/eme_if]
[eme_if tag='#_IS_REGISTERED_PENDING' value='1']<span class="waitingcancellink"><a href="#_EVENTPAGEURL">Liste attente</a></span>[/eme_if]
[eme_if tag='#_IS_REGISTERED' value='0']<a class="registerlink" href="#_EVENTPAGEURL">S'inscrire</a>[/eme_if]</strong></li>
```

The problem is that #_IS_REGISTERED_APPROVED return 1 even if the users is on waiting list
And #_IS_REGISTERED_PENDING is not triggered if the user is on waiting list.
[eme_if tag='#_IS_REGISTERED' value='0'] is working fine :)

Any idea ?

Thanks

**Comment by liedekef** [comment]
To be correct: there's no such thing as a user being on the waiting list, but a booking.
Next to that, can you check the status of these bookings in the DB? Because normally all bookings on the waiting list are pending bookings as well. How were these bookings made? From the frontend or the backend?

**Reply by MichaelWybo** [reply]
Booking are made frontend, and in fact it's the booking that i check :)

Has shown here : https://www.e-dynamics.be/wordpress/conditional-tags/
I was hoping the differenciate those on waiting list.

![image](https://github.com/liedekef/events-made-easy/assets/32988994/f6ea3bfd-4fe0-495c-b789-4a542616ac2d)

[eme_if tag='#_IS_REGISTERED' value='0'] is working fine :)
[eme_if tag='#_IS_REGISTERED' value='1'] is working fine :)

But as I wrote just before
The problem is that #_IS_REGISTERED_APPROVED return 1 even if the user's booking is on waiting list
And #_IS_REGISTERED_PENDING is not triggered if the user's booking is on waiting list.

**Reply by liedekef** [reply]
#_IS_REGISTERED_APPROVED is 1 if the user has no pending bookings at all
I'll test here the 2 placeholders too, but I think you need to check the state of each booking, not the case if a user has pending bookings or not. See the RSVP placeholders #_ON_WAITINGLIST, #_IS_PENDING, #_IS_APPROVED etc ...

**Reply by liedekef** [reply]
ok, I found a typo in the code. This fix should work:
https://github.com/liedekef/events-made-easy/commit/65703ef080c1de2061697a75914564a9e463042a

**Reply by MichaelWybo** [reply]
Working !!! Perfect as usual

Thnaks a lot
